src/ECL-1.0.xml

```diff
@@ -5,21 +5,19 @@
       <crossRefs>
          <crossRef>http://opensource.org/licenses/ECL-1.0</crossRef>
       </crossRefs>
-      <standardLicenseHeader> Copyright (c) 
-    <alt match=".+" name="copyright">&lt;year&gt; &lt;copyright holders&gt;</alt> Licensed under the Educational
```

Comment:
This markup was missing `<p>`, so copyright holders was running into the “Licensed under…” sentence without intervening punctuation.  The markup inside `<text>` is already using separate paragraphs.